src/main/java/com/google/devtools/build/lib/bazel/repository/skylark/SkylarkRepositoryContext.java

```diff
@@ -487,6 +492,32 @@ private void validateExecuteArguments(Sequence<?> arguments) throws EvalExceptio
     }
   }
 
+  /** Returns the command line arguments as a string for display in the profiler. */
+  private static String profileArgsDesc(String method, List<String> args) {
+    StringBuilder b = new StringBuilder();
+    b.append(method).append(": ");
+
+    String sep = "";
+    for (String arg : args) {
+      int appendLen = arg.length() + sep.length();
```

Comment:
super-nit: can you please swap the `arg` and `sep` here to have the same order as the appending order?

Reply:
done